arangod/GeneralServer/RestHandler.cpp

```diff
@@ -113,7 +113,20 @@ futures::Future<Result> RestHandler::forwardRequest(bool& forwarded) {
     return futures::makeFuture(Result());
   }
 
-  std::string serverId = forwardingTarget();
+  ResultT forwardResult = forwardingTarget();
+  if (forwardResult.fail()) {
+    return futures::makeFuture(forwardResult.result());
+  }
+
+  auto forwardContent = forwardResult.get();
```

Comment:
maybe we can do
```suggestion
  auto const [serverId, removeHeader] = forwardResult.get();
```
Here, but this is not urgent, and will not block this PR

Reply:
Do not commit this, from UI.
Needs modification of the lines below.

Reply:
First I've implemented with auto. But a lambda cb had issues with it, Row: 199:
```
 auto cb = [this, serverId, useVst,
             self = shared_from_this()](network::Response&& response) -> Result {
```